flexmeasures/cli/data_add.py

```diff
@@ -1638,6 +1215,22 @@ def add_schedule(  # noqa C901
             click.secho("New schedule is stored.", **MsgStyle.SUCCESS)
 
 
+@fm_add_data.group(
+    "schedule",
+    cls=DeprecatedDefaultGroup,
+    default="storage",
+    deprecation_message="The command 'flexmeasures add schedule' is deprecated. Please use `flexmeasures add schedule for-storage` instead.",
+)
+@click.pass_context
+@with_appcontext
+def create_schedule(ctx):
+    """(Deprecated) Create a new schedule for a given power sensor.
+
+    THIS COMMAND HAS BEEN RENAMED TO `flexmeasures add schedule for-storage`
+    """
+    pass
+
+
 @create_schedule.command("for-process", cls=DeprecatedOptionsCommand)
```

Comment:
This can also go, and we should use the new `add schedule` command in its place (update the tutorial that deals with processes).

I believe that all it does is to wrap the custom parameters into a custom flex-model and flex-context. By making sure the ProcessScheduler is handling the schedule, the tutorial should work just as well.